tox.ini

```diff
@@ -16,13 +16,13 @@ commands = pytest {posargs}
 description = check the code style
 basepython = python3.6
 commands =
-    flake8 {toxinidir}/edgePy
-    pylint {toxinidir}/edgePy --errors-only --output-format=colorized
+    flake8 {toxinidir}/src
+    pylint {toxinidir}/src --errors-only --output-format=colorized
```

Comment:
I would much prefer us to go back to naming the top-level dir `edgePy` as it best practice in developing a Python package. Thanks!